parser/importer_pytesseract.py

```diff
@@ -0,0 +1,170 @@
+# Python3
```

Comment:
Can we use `#!/usr/bin/env python3` here?
Here's an explanation: https://stackoverflow.com/a/19305076/270334